ui/packages/components/src/DatePicker/RangePicker.tsx

```diff
@@ -0,0 +1,314 @@
+      <PopoverContent>
+        <div className="flex flex-row bg-white">
+          <div className={`${showAbsolute && 'min-h-[589px]'} w-[250px] border-r border-slate-300`}>
+            <div className="m-2">
+              <Input
+                ref={durationRef}
+                type="text"
+                placeholder="Enter Relative Time (5s, 1m, 2d)"
+                minLength={2}
+                maxLength={64}
+                error={durationError}
+                onKeyDown={processDuration}
+              />
+            </div>
+            {Object.entries(RELATIVES).map(([k, v], i) => {
+              const planValid = validateDuration(k, upgradeCutoff);
+              return (
+                <div
+                  key={`duration-${i}`}
+                  className={`flex flex-row items-center justify-between py-2 pl-6 pr-3 text-sm font-normal text-slate-700 hover:bg-blue-50 ${
+                    planValid ? 'cursor-pointer' : 'cursor-not-allowed'
+                  }`}
+                  {...(planValid && {
+                    onClick: () => {
+                      setDisplayValue(<RelativeDisplay duration={k} />);
+                      onChange({ type: 'relative', duration: parseDuration(k) });
+                      setDurationError('');
+                      setOpen(false);
+                    },
+                  })}
+                >
+                  {v}
+                  {!planValid && (
+                    <Badge
+                      className="border-indigo-500 px-2 py-0.5 text-xs text-indigo-500"
+                      kind="outlined"
+                    >
+                      Upgrade Plan
+                    </Badge>
+                  )}
+                </div>
+              );
+            })}
+            <div className="flex flex-col">
+              <div
+                className={`cursor-pointer border-t border-slate-300 px-6 py-3.5 text-sm font-normal text-slate-700 hover:bg-blue-50 ${
+                  showAbsolute && 'bg-blue-50'
+                }`}
+                onClick={() => {
+                  setShowAbsolute(!showAbsolute);
+                  setDurationError('');
+                }}
+              >
+                Absolute Range
+              </div>
+              {showAbsolute && (
+                <div className="px-[22px] py-2 text-[13px] font-normal text-slate-500">
+                  {formatAbsolute(absoluteRange)}
+                </div>
+              )}
+            </div>
+          </div>
+          {showAbsolute && (
+            <div className="flex w-[354px] flex-col">
+              <Tabs.Root className="flex flex-col" value={tab} onValueChange={setTab}>
+                <Tabs.List
+                  className="border-mauve6 flex shrink-0 border-b"
+                  aria-label="Manage your account"
+                >
+                  <Tabs.Trigger
+                    className="flex h-[52px] flex-1 cursor-pointer select-none items-center 
```

Comment:
Could we use h-11 instead? It's more standard, I know the design has more, but it's unnecessary vertical space that creates more need for a scroll. For some users it might be the difference between having to scroll or not, or even realizing that there's a button at the bottom of the picker

Reply:
👍